charts/kyverno-policies/templates/pod-sec-ctxt-capmc.yaml

```diff
@@ -0,0 +1,42 @@
+{{- $name := "pod-sec-ctxt-capmc" }}
+{{- if eq (include "kyverno-policy.podSecurityBaseline" (merge (dict "name" $name) .)) "true" }}
+{{- include "kyverno-policy.supportedKyvernoCheck" (dict "top" . "ver" ">= 1.6.0-0") }}
+apiVersion: kyverno.io/v1
+kind: Policy
+metadata:
+  name: {{ $name }}
+  namespace: services
+  annotations:
+    policies.kyverno.io/title: Mutate pod and container security contexts
+    policies.kyverno.io/category: Pod Security Standards (Baseline)
+    {{- if .Values.podSecuritySeverity }}
+    policies.kyverno.io/severity: {{ .Values.podSecuritySeverity }}
+    {{- end }}
+    kyverno.io/kyverno-version: 1.6.0
+    policies.kyverno.io/minversion: 1.6.0
+    policies.kyverno.io/subject: Pod
+    policies.kyverno.io/description: >-
+      Highly restrictive mutate policy setting container security context
+spec:
+  rules:
+  - name: set-container-security-context
+    match:
+      resources:
+        kinds:
+        - Pod
+        selector:
+          matchLabels:
+            app.kubernetes.io/instance: cray-hms-capmc
+            app.kubernetes.io/name: cray-capmc
+    mutate:
+      foreach:
+      - list: "request.object.spec.containers"
+        patchStrategicMerge:
+          spec:
+            containers:
+            - (name): |-
+                {{`{{element.name}}`}}
+              securityContext:
+                +(allowPrivilegeEscalation): false
```

Comment:
The +() only sets the tag if it is not present, https://kyverno.io/docs/writing-policies/mutate/#conditional-logic-using-anchors. This is not the behavior we want. I believe you'll want to remove the conditional logic. 

For non-exceptions, a workload is allowed to override runAsGroup and runAsUser. No override will be allowed for runAsNonRoot (true), allowPrivilegeEscalation (false), nor privileged (false), unless by documented exception (and there shouldn't be any in this PR).

Reply:
okay